### PR TITLE
Update New feature 'Sample Helm Chart for Axway Managed Cloud'

### PR DESCRIPTION
**Pull request by jowens-axway**
testing out adding changes to via the github ui

Thank you for your contribution to the Axway-Open-Docs repo.

## Describe the changes

Enter a brief description of your changes to communicate to the reviewers what you changed and why.

## Deploy preview link

Please add the deploy preview link to the **specific page** that you've changed.

_Although Netlify will add this link below, it will redirect to the main page of the documentation, not to an specific page._

## Checklist for contributors

Before submitting this PR, please make sure:

* [ ] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [ ] You have signed the [Axway CLA](https://cla.axway.com/)
* [ ] You have verified the technical accuracy of your change
* [ ] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [ ] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)
* [ ] You have verified that all status checks have passed

_Put an x in the boxes that apply. This is simply a reminder of what we are going to look for before merging your change._

### CODE REVIEW

**Review by dginevaAxway — APPROVED** [review]

File: content/en/docs/apim_relnotes/20210330_apip_relnotes.md [hunk 1]
```diff
@@ -19,14 +19,10 @@ API Portal is available as a software installation or a virtualized deployment i
...
 
-### placeholder 1
-
-placeholder text
-
-For more information, see:
-
-* [some reference 1](/docs/placeholder)
-* [some reference 2](/docs/placeholder)
+### Sample Helm Chart for Axway Managed Cloud (Internal release only)
```
Comment (andreamussap) [inline content/en/docs/apim_relnotes/20210330_apip_relnotes.md:22]: What's this sample helm chart?
Why/ How to use it?
Is it documented somewhere? (For more information, see...)
Is this PR related to a Jira ticket?

**Comment by andreamussap** [comment]
Duplicated by #1700
